internal/pkg/config/config_test.go

```diff
@@ -63,10 +103,17 @@ func TestReadConfig(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, err := ReadConfig(tt.args.filePath)
-			if (err != nil) != tt.wantErr {
+			if err == nil && tt.wantErr {
 				t.Errorf("ReadConfig() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+			if err != nil && tt.wantErr && tt.errValue != nil {
```

Comment:
This condition doesn't cover the case where we didn't want an error i.e `tt.wantErr==false` and yet we got an error.

Reply:
I don't understand this. Since we are defining the expectation how can we say that we do not want an error but error comes? Because if error comes we have that covered in want error true cases